src/main/java/com/synopsys/integration/detect/configuration/DetectConfigurationFactory.java

```diff
@@ -111,7 +111,7 @@ public Long findTimeoutInSeconds() {
             Long timeout = getValue(DetectProperties.DETECT_API_TIMEOUT);
             return timeout / 1000;
         } else {
-            return getValue(DetectProperties.DETECT_REPORT_TIMEOUT);
```

Comment:
Shouldn't we add
```
else if(detectConfiguration.wasPropertyProvided(DetectProperties.DETECT_REPORT_TIMEOUT) {
    return getValue(DetectProperties.DETECT_REPORT_TIMEOUT);
} else {
    return getValue(DetectProperties.DETECT_TIMEOUT);
}
```
Without the else if the user can set the DETECT_REPORT_TIMEOUT and it is never used.  Being that it is deprecated shouldn't it take precedence over the new property.